sevntu-checks/src/main/java/com/github/sevntu/checkstyle/checks/coding/ForbidCertainMethodsLoader.java

```diff
@@ -0,0 +1,151 @@
+            throws ParserConfigurationException, SAXException {
+        super(Collections.singletonMap(DTD_PUBLIC_ID_1_0, DTD_RESOURCE_NAME_1_0));
+    }
+
+    @Override
+    public void startElement(String namespaceUri,
+                             String localName,
+                             String qName,
+                             Attributes attributes)
+            throws SAXException {
+        if ("ForbiddenMethod".equals(qName)) {
+            //add SuppressElement filter to the filter chain
+            final String methodName = attributes.getValue("methodName");
+            final String argCount = attributes.getValue("argCount");
+            if (methodName == null) {
+                // -@cs[IllegalInstantiation] SAXException is in the overridden method signature
+                throw new SAXException("missing methodName attribute");
+            }
+            if (argCount == null) {
+                // -@cs[IllegalInstantiation] SAXException is in the overridden method signature
+                throw new SAXException("missing argCount attribute");
+            }
+            // ensure that regex is valid
+            Pattern.compile(methodName);
+            Pattern.compile(argCount);
+            callInfoList.add(ForbidCertainMethodsCheck.CallInfo.of(methodName, argCount));
+        }
+    }
+
+    /**
+     * Returns the suppression filters in a specified file.
+     *
+     * @param filename name of the suppression file.
+     * @return the filter chain of suppression elements specified in the file.
+     * @throws CheckstyleException if an error occurs.
+     */
+    static List<ForbidCertainMethodsCheck.CallInfo> loadConfigValue(String filename)
+            throws CheckstyleException {
+        final URI uri = CommonUtils.getUriByFilename(filename);
+        final InputSource source = new InputSource(uri.toString());
+        return loadConfigValue(source, filename);
+    }
+
+    /**
+     * Returns the suppression filters in a specified source.
+     *
+     * @param source     the source for the suppressors.
+     * @param sourceName the name of the source.
+     * @return the filter chain of suppression elements in source.
+     * @throws CheckstyleException if an error occurs.
+     */
+    private static List<ForbidCertainMethodsCheck.CallInfo> loadConfigValue(InputSource source,
+            String sourceName) throws CheckstyleException {
+        try {
+            final ForbidCertainMethodsLoader suppressionsLoader = new ForbidCertainMethodsLoader();
+            suppressionsLoader.parseInputSource(source);
+            return suppressionsLoader.callInfoList;
+        }
+        catch (final FileNotFoundException ex) {
+            throw new CheckstyleException(UNABLE_TO_FIND_ERROR_MESSAGE + sourceName, ex);
+        }
+        catch (final ParserConfigurationException | SAXException ex) {
+            final String message = String.format(Locale.ROOT, "Unable to parse %s - %s",
```

Comment:
We already have an issue to disable `Locale.ROOT` so I don't think we should be using it here.
See #598 .

Reply:
Will remove it.